pico/client.py

```diff
@@ -10,8 +10,8 @@
 
 Use help(example.hello) or example.hello? as normal to check function parameters and docstrings.
 """
-
-import pico.pragmaticjson as json
+import os
+from . import pragmaticjson as json
```

Comment:
it would be better not to use dot prefixes on imports for being explicit and to understand context better